force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -176,11 +214,11 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String getCurrentDate() {
         // Get the todays date
-
+        Date d = Date.today();
```

Comment:
Try to avoid using single-character variable names (eg, `d`). `dateToday `would work here. The only time this is acceptable is inside a traditional for loop: `(Integer i=0; i<n; i++)`

Reply:
Agree. Was being lazy on that one.